controllers/dashboardController.js

```diff
@@ -48,7 +76,10 @@ module.exports.dashboard = async function dashboard(req, res) {
 
     // @TODO create a function for this
     const patientsWithFolderStatus = patients.map ( patient => {
-      patient.hasFolderCompleted = hasFolderCompleted(patient)
+      const {folderCompleted, missingInfo} = hasFolderCompleted(patient)
+      patient.hasFolderCompleted = folderCompleted
+      patient.missingInfo = missingInfo
+
```

Comment:
separate this in two functions, one for the patients info and one for the global hasAllPatientsWithCompletedFolders variable